app/controllers/reviews_controller.rb

```diff
@@ -0,0 +1,19 @@
+class ReviewsController < ApplicationController
+
+  def new
+    @shelter = Shelter.find(params[:shelter_id])
+  end
+
+  def create
+    shelter = Shelter.find(params[:shelter_id])
+    shelter.reviews.create(review_params)
```

Comment:
create! changed to create

Reply:
do you know if we need to add a .save as well when you lose the bang? Also, let's make a note to add conditional globally based on what we just learned...

Reply:
create includes both... thanks @mikez321